specification/storagecache/resource-manager/readme.md

```diff
@@ -101,8 +112,6 @@ See configuration in [readme.typescript.md](./readme.typescript.md)
 
 See configuration in [readme.ruby.md](./readme.ruby.md)
```

Comment:
Need to be updated to reflect the 2020-10-01 version, you can check the existing blocks in the readme.

Reply:
Actually for below languages.
Go
Java
Ruby
AzureResourceSchema

Reply:
Fixed in next commit.

Reply:
Fixed in next commit.